watcher/watcher.go

```diff
@@ -461,12 +463,19 @@ func (w *Watcher) ReportRoomsStatuses() error {
 	}
 
 	for _, r := range roomDataSlice {
-		_ = reporters.Report(reportersConstants.EventGruStatus, map[string]interface{}{
+		err := reporters.Report(reportersConstants.EventGruStatus, map[string]interface{}{
 			reportersConstants.TagGame:      w.GameName,
 			reportersConstants.TagScheduler: w.SchedulerName,
 			"status":                        r.Status,
 			"gauge":                         r.Gauge,
 		})
+		if err != nil {
+			w.Logger.Error(err)
```

Comment:
This is the most important error to show. We need to find it by a tag, so can you change it to this?
```suggestion
			w.Logger.WithError(err).Error("failed to report room status")
```

Reply:
Oh yeah! I just have fixed it some minutes ago 😅